careerfairsystems/arkad-app-2018
# Add real coordinators

## Change description

- Add real coordinators
- Move FAQ button to header

## How to verify

- Go to the "About" screen and press the "The ARKAD team" button
  - Verify that the correct image is displayed with the correct name and role 
- Go to the "About" screen
  - Verify that the FAQ button is in the header and that clicking it takes you to the FAQ screen

## Issues fixed

This fixes #85, fixes #88.
